src/main/java/org/jabref/logic/importer/fileformat/EndnoteXmlImporter.java

```diff
@@ -260,10 +260,32 @@ private EntryType getType(Record record) {
     }
 
     private Optional<String> getUrlValue(Url url) {
-        return Optional.ofNullable(url)
-                       .map(Url::getStyle)
-                       .map(Style::getContent)
-                       .map(this::clean);
+        Optional<List<Object>> urlContent = Optional.ofNullable(url).map(Url::getContent);
+        List<Object> list = urlContent.orElse(null);
```

Comment:
```suggestion
        List<Object> list = urlContent.orElse(Collections.emptyList());
```
This has the advantage that you can omit the null check and just check for empty

Reply:
Yes, indeed. It can also avoid some underlying mistakes. Already changed.